lib/oli_web/live/system_message_live/index_view.ex

```diff
@@ -100,8 +116,8 @@ defmodule OliWeb.SystemMessageLive.IndexView do
     new_message_attrs =
       attrs
       |> Oli.Utils.atomize_keys()
-      |> Map.update(:start, "", &datestring_to_utc_datetime(&1, socket.assigns.local_tz))
-      |> Map.update(:end, "", &datestring_to_utc_datetime(&1, socket.assigns.local_tz))
+      |> Map.update(:start, "", &datestring_to_utc_datetime(&1, socket.assigns.local_timezone))
```

Comment:
There are some facilities that exist to format datetimes in the current user's timezone, then fallback to UTC if not is not available. See `OliWeb.Common.FormatDateTime.date`.

This helper function is already imported into all live views in `OliWeb` so yo should be able ot directly call `date(some_date, @context)` where @context is an assign set by calling `context = OliWeb.Common.SessionContext.init(session)` in the mount. See `lib/oli_web/live/qa/qa_live.ex` for an example

Reply:
Ah, nevermind. I now realize this is already utilizing this